pom.xml

```diff
@@ -114,8 +114,8 @@
     </dependency>
     <dependency>
       <groupId>org.kohsuke</groupId>
-      <artifactId>asm5</artifactId>
-      <version>5.0.1</version>
+      <artifactId>asm6</artifactId>
```

Comment:
Downstream of something in our repackaging?

Reply:
Yes @jglick https://github.com/kohsuke/package-renamed-asm/pull/3 released since by Kohsuke.

Reply:
Yes @jglick https://github.com/kohsuke/package-renamed-asm/pull/3 released since by Kohsuke.